include/utility/json_utility.hpp

```diff
@@ -550,5 +550,59 @@ inline std::string getFruPathFromJson(const nlohmann::json& i_sysCfgJsonObj,
     }
     return std::string();
 }
+
+/**
+ * @brief Get inventory object path from system config JSON.
+ *
+ * Given either D-bus inventory path/FRU EEPROM path/redundant EEPROM path,
```

Comment:
what's the difference between D-bus inventory path and D-bus inventory object path ?

Reply:
no difference :) modified